src/Umbraco.Tests/Persistence/Repositories/LanguageRepositoryTest.cs

```diff
@@ -74,7 +73,7 @@ public void Can_Perform_Get_By_Iso_Code_On_LanguageRepository()
                 // Assert
                 Assert.That(language, Is.Not.Null);
                 Assert.That(language.HasIdentity, Is.True);
-                Assert.That(language.CultureName, Is.EqualTo(au.DisplayName));
+                Assert.That(language.CultureName, Is.EqualTo(au.EnglishName));
```

Comment:
The `DisplayName` can differ based on current UI thread culture, set let's use `EnglishName` to keep the assertion/error message consistent.